frontend/src/Pages/UserPage.tsx

```diff
@@ -88,6 +88,13 @@ export const UserPage = observer(() => {
             refreshProfile();
         };
 
+        const daysAgoToStr = (daysAgo: number) => {
+            const titles = ['день', 'дня', 'дней'];
+            const cases = [2, 0, 1, 1, 1, 2];
+            return daysAgo === 0 ? 'сегодня'
+                : `${daysAgo} ${titles[daysAgo % 100 > 4 && daysAgo % 100 < 20 ? 2 : cases[daysAgo % 10 < 5 ? daysAgo % 10 : 5]]} назад`;
```

Comment:
<img width="221" alt="image" src="https://github.com/spaceshelter/orbitar/assets/2865203/c8009363-1a10-4c20-b6fe-8e804cdca51e">

Two notes:
1. Multiline text breaks the layout of the tooltip
2. Precise number of days makes little sense for anything over ≈5. Consider using 
`moment.duration.humanize`:

```
`${moment.duration(daysAgo, 'days').humanize(true)} назад`
```
with different threshold for days, weeks, months, years

Reply:
both 1 & 2 fixed
![image](https://github.com/spaceshelter/orbitar/assets/3017441/1ce97a5b-247c-483a-8cd7-e7ca4539699c)